exercises/practice/acronym/.meta/Acronym.example.ps1

```diff
@@ -0,0 +1,22 @@
+Function Get-Acronym() {
+    <#
+    .SYNOPSIS
+    Get the acronym of a phrase.
+
+    .DESCRIPTION
+    Given a phrase, return the string acronym of that phrase.
+    "As Soon As Possible" => "ASAP"
+    
+    .PARAMETER String
```

Comment:
This should say `.PARAMETER Phrase`

Reply:
I see what you meant on the forum now, I was focus on the code part and forgot about this bit